packages/controller-utils/src/util.ts

```diff
@@ -299,8 +295,6 @@ function toChecksumHexAddressUnmemoized(address: string): string;
  * and is only present for backward compatibility. It may be removed in a future
  * major version. Please pass a string to `toChecksumHexAddress` instead.
  */
-// TODO: Either fix this lint violation or explain why it's necessary to ignore.
-// eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
Wonder why this isn't causing an error anymore 🤔 I think we intend to still disallow one letter generics, right?

Reply:
It's disabled in our config for certain files. Something to revisit in a later PR